code/modules/vote/vote_presets.dm

```diff
@@ -0,0 +1,46 @@
+// Crew transfer vote
+/datum/vote/crew_transfer
+	question = "End the shift"
+	choices = list("Initiate Crew Transfer", "Continue The Round")
+	vote_type_text = "crew transfer"
+
+/datum/vote/crew_transfer/New()
+	if(SSticker.current_state < GAME_STATE_PLAYING)
+		CRASH("Attempted to call a shuttle vote before the game starts!")
+	..()
+
+/datum/vote/crew_transfer/handle_result(result)
+	if(!(result in choices))
+		CRASH("Result for a crew transfer vote wasnt a valid result. What the heck?")
```

Comment:
I feel like this check should be performed in the parent, since you'd want it in any vote, not just crew transfer.